### PR TITLE
Release v0.5.30

### PR DESCRIPTION
**Pull request by thomshutt**
**What does this pull request do? Explain your changes. (required)**

Version bump to v0.5.30.

**Specific updates (required)**

See commit history.

**How did you test each of these updates (required)**

**Does this pull request close any open issues?**

**Checklist:**

- [x] Read the [contribution guide](./doc/contributing.md)
- [x] `make` runs successfully
- [x] All tests in `./test.sh` pass
- [x] README and other documentation updated
- [x] [Pending changelog](./CHANGELOG_PENDING.md) updated

### CODE REVIEW

**Review by yondonfu — APPROVED** [review]
LGTM.

Looks like we can rebase on top of master since https://github.com/livepeer/go-livepeer/pull/2390 is now merged.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,61 @@
...
+- [#2289](https://github.com/livepeer/go-livepeer/pull/2289) Add timeouts to ETH client (@leszko)
+- [#2282](https://github.com/livepeer/go-livepeer/pull/2282) Add checksums and gpg signature support with binary releases. (@hjpotter92)
+- [#2344](https://github.com/livepeer/go-livepeer/pull/2344) Use T.TempDir to create temporary test directory (@Juneezee)
+- [#2353](https://github.com/livepeer/go-livepeer/pull/2353) Codesign and notarize macOS binaries to be allowed to run without warnings on apple devices (@hjpotter92)
+- [#2351](https://github.com/livepeer/go-livepeer/pull/2351) Refactor livepeer.go to enable running Livepeer node from the code (@leszko)
+- [#2372](https://github.com/livepeer/go-livepeer/pull/2372) Upload test coverage reports to codecov (@hjpotter92)
+
+#### Broadcaster
+- [#2309](https://github.com/livepeer/go-livepeer/pull/2309) Add dynamic timeout for the orchestrator discovery (@leszko)
+- [#2337](https://github.com/livepeer/go-livepeer/pull/2337) Fix dynamic discovery timeout to not retry sending requests, but wait for the same request to complete (@leszko)
```
Comment (leszko) [inline CHANGELOG.md]: I know we always have one bullet point per PR, but looking at it from the user perspective, the following 3 PRs provides one functionality: "Add dynamic timeout for the orchestrator discovery":
- #2309 => The actual PR introducing a feature
- #2337 => Fix for the PR above
- #2296 => Attempt before the above was implemented

It's not strictly related to this release, but I think we can consider creating a changelog which is more user facing and less PR-driven.

Saying that, we can keep it as it is for this release.

File: cmd/livepeer_cli/wizard.go [hunk 2]
```diff
@@ -307,7 +307,7 @@ func httpPostWithParamsHeaders(url string, val url.Values, headers map[string]st
 
 	defer resp.Body.Close()
 	result, err := ioutil.ReadAll(resp.Body)
-	if err != nil || string(result) == "" {
```
Comment (leszko) [inline cmd/livepeer_cli/wizard.go]: I think you need to rebase and this change should not be visible in the release PR.

**Review by leszko — APPROVED** [review]
Added 2 comments. Other than that, LGTM 👍

File: cmd/livepeer_cli/wizard.go (hunk 2 above)
Comment (thomshutt) [reply cmd/livepeer_cli/wizard.go]: That's weird, I cherry-picked your commit onto the release branch. Will do a rebase

File: CHANGELOG.md (hunk 1 above)
Comment (thomshutt) [reply CHANGELOG.md]: Good point, I've kept the first one and removed the other two

**Pull request merged**